src/Groups/GAPGroups.jl

```diff
@@ -700,6 +700,9 @@ julia> minimal_generating_set(symmetric_group(5))
    return res
 end
 
+# keep the perhaps misleading old name
+minimal_generating_set(G::GAPGroup) = minimal_size_generating_set(G)
```

Comment:
I would vote for this getting a proper deprecation, i.e. instead of this line here add
```julia
@deprecate minimal_generating_set(G::GAPGroup) minimal_size_generating_set(G)
```
to `deprecations.jl`

Reply:
O.k., I will change this. (My interpretation of Max' comment had been that we just keep the old name.)